charts/consul/values.yaml

```diff
@@ -29,6 +29,13 @@ global:
   # Consul into Kubernetes will have, e.g. `service-name.service.consul`.
   domain: consul
 
+  # Configures the Cluster Peering feature.
```

Comment:
Should we mark this with [TECH PREVIEW] or something?

Reply:
i added [Experimental] for this and the CRD fields

Reply:
would [Tech Preview] make more sense? I thought it might be a more internal word?

Reply:
yup!! :)